src/main/kotlin/com/example/olaf/blogtutorial/HtmlController.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.olaf.blogtutorial
+
+import org.springframework.stereotype.Controller
+import org.springframework.ui.Model
+import org.springframework.ui.set
+import org.springframework.web.bind.annotation.GetMapping
+
+@Controller
+class HtmlController{
+    @GetMapping("/")
+    fun blog(model: Model): String{
```

Comment:
```suggestion
class HtmlController {
    @GetMapping("/")
    fun blog(model: Model): String {
```
그리고 지금 파일들 끝에 
![스크린샷 2020-03-19 오전 9 50 23](https://user-images.githubusercontent.com/33410105/77020436-129e6400-69c7-11ea-95fe-99299f9c9e6f.png)
요렇게게 되어있자나요?
이게 개행을 하라는 거에용
한줄씩 엔터 쳐서 푸시 하시면 아마 없어질거에요

Reply:
오..넵 수정하겠습니다 😄